evap/results/tools.py

```diff
@@ -170,43 +169,45 @@ def _calculate_results_impl(course):
 
     return sections
 
-
-def calculate_average_grades_and_deviation(course):
-    """Determines the final average grade and deviation for a course."""
+def calculate_average_grade(course):
     avg_generic_likert = []
     avg_contribution_likert = []
-    dev_generic_likert = []
-    dev_contribution_likert = []
     avg_generic_grade = []
     avg_contribution_grade = []
-    dev_generic_grade = []
-    dev_contribution_grade = []
 
     for __, contributor, __, results, __ in calculate_results(course):
         average_likert = avg([result.average for result in results if result.question.is_likert_question])
-        deviation_likert = avg([result.deviation for result in results if result.question.is_likert_question])
         average_grade = avg([result.average for result in results if result.question.is_grade_question])
-        deviation_grade = avg([result.deviation for result in results if result.question.is_grade_question])
 
         (avg_contribution_likert if contributor else avg_generic_likert).append(average_likert)
-        (dev_contribution_likert if contributor else dev_generic_likert).append(deviation_likert)
         (avg_contribution_grade if contributor else avg_generic_grade).append(average_grade)
-        (dev_contribution_grade if contributor else dev_generic_grade).append(deviation_grade)
 
     # the final total grade will be calculated by the following formula (GP = GRADE_PERCENTAGE, CP = CONTRIBUTION_PERCENTAGE):
     # final_likert = CP * likert_answers_about_persons + (1-CP) * likert_answers_about_courses
     # final_grade = CP * grade_answers_about_persons + (1-CP) * grade_answers_about_courses
     # final = GP * final_grade + (1-GP) * final_likert
```

Comment:
do you think this comment helps? to me it it's actually harder to understand as the code below.